site/content/en/docs/Installation/_index.md

```diff
@@ -62,18 +58,9 @@ The `01-open-match-core.yaml` contains:
 * ServiceAccounts, Roles, and RoleBindings to define Open Match deployments' IAMs.
 * HorizontalAutoScalars to auto-scale Open Match based on pods' average CPU utilization.
 
-Here are the commands to install the Open Match core in your cluster:
-
-```bash
-# Create a cluster role binding (if using gcloud on Linux or OSX)
-kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user `gcloud config get-value account`
```

Comment:
Just to confirm these steps are not required since k8s 1.13, right? Perhaps we should add a prerequisite section in this guide and point out the support golang and k8s version.

Reply:
Lets tackle that in the issue that explicitly tracks stating kubernetes version.